### ISSUE TITLE
Bug: Start of add-on fails because s6-rc failed

### ISSUE THREAD

**Issue by AlexKnowsIt** [BenoitAnastay/paperless-home-assistant-addon#210]
First, thank you for bringing paperless to homeassistant.
Currently I am running paperless in a separate container but would prefer to manage everything from inside home-assistant.
I am trying for the last three days to bring up the add-on in my homeassistant but somehow I'm stuck. The installation takes quite some time but seems to work. When I want to start the add-on I get the following error message:
`/run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.`
Sadly I don't get to more information because I can't access the inside of the docker container inside of hasos.
I am running homeassistant on a x64 machine on hasos. Everything is up to date.

**Comment by AlexKnowsIt** [comment]
Here the rest of the logs. Do you have an idea if there is a bad configuration on my part or if the configuration problem is in the add-on config?
`[34m Add-on version: 2.1.8[0m
[32m You are running the latest version of this add-on.[0m
[34m System: Home Assistant OS 14.0  (amd64 / qemux86-64)[0m
[34m Home Assistant Core: 2024.12.3[0m
[34m Home Assistant Supervisor: 2024.11.4[0m
[34m-----------------------------------------------------------[0m
[34m Please, share the above information when looking for help[0m
[34m or support in, e.g., GitHub, forums or the Discord chat.[0m
[34m-----------------------------------------------------------[0m
s6-rc: info: service base-addon-banner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service base-addon-timezone: starting
s6-rc: info: service base-addon-log-level: starting
s6-rc: info: service fix-attrs successfully started
[20:41:51] INFO: [32mConfiguring timezone (Europe/Berlin)...[0m
s6-rc: info: service base-addon-log-level successfully started
s6-rc: info: service base-addon-timezone successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service init-paperless: starting
s6-rc: info: service init-nginx: starting
s6-rc: info: service redis: starting
s6-rc: info: service redis successfully started
[34mIngress is enabled[0m
[20:41:56] ERROR: [35mPlease change default admin password before running addon[0m
s6-rc: warning: unable to start service init-paperless: command exited 1
s6-rc: info: service init-nginx successfully started
/run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.
/run/s6/basedir/scripts/rc.init: fatal: stopping the container.
`

**Comment by BenoitAnastay** [comment]
It's probably because you haven't personalised your configuration and haven't set an admin password

**Comment by AlexKnowsIt** [comment]
The weird thing is that I changed the password. It seems that the configuration is not used in the add-on. Maybe something was missing in the initial install. Can you make sense of the supervisor log? Is this a temp folder?
I uninstalled and reinstalled the add-on already but the error message stays the same. I am already thinking about resetting my whole ha and trying the add-on in a fresh installation.

**Comment by BenoitAnastay** [comment]
I don't think that you need to reset home assistant but my script must be aborted if the password is changeme in the addon configuration tab

**Comment by AlexKnowsIt** [comment]
Thank you. After a reinstall of the addon I changed first the credentials. Now everything is working like a charm!

**Comment by AlexKnowsIt** [comment]
I have one more question. Did you find the possibility to use the management utilities inside of ha? https://docs.paperless-ngx.com/administration/#management-commands
I tried to run the `document-renamer`
Kind regards

**Comment by BenoitAnastay** [comment]
It works using `docker exec`

**Comment by BenoitAnastay** [comment]
@AlexKnowsIt Thanks a lot for the sponsorship !

**Comment by AlexKnowsIt** [comment]
You're very welcome. Your work for this addon is very good. Continue the good work 🙏
